### PR TITLE
First and last iteration

### PR DESCRIPTION
**Pull request by tayloris**
I change the template file for webiz to make posible to plot the firs and last itration in webiz

### CODE REVIEW

File: src/flownet/templates/webviz_ahm_config.yml.jinja2 [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Flownet
 
 shared_settings:
   scratch_ensembles: 
-  {%- for iteration in iterations[-1:] %}
+  {%- for iteration in [iterations[0],iterations[-1]] %}
```
Comment (anders-kiaer) [inline src/flownet/templates/webviz_ahm_config.yml.jinja2:7]: Add a space after `,`? Same for the other changes. Otherwise LGTM 🎉

**Comment by tayloris** [comment]
I add the spaces but still it didnt pass LGTM

**Comment by wouterjdb** [comment]
Seems like it is still running; but don't forget to click the "update branch" button (or do it manually on your side). Because before the branch is up to date you can't merge (and then check will be run again anyways)

File: src/flownet/templates/webviz_ahm_config.yml.jinja2 [hunk 2]
```diff
@@ -4,7 +4,7 @@ title: Flownet
 
 shared_settings:
   scratch_ensembles: 
-  {%- for iteration in iterations[-1:] %}
+  {%- for iteration in [iterations[0] , iterations[-1]] %}
```
Comment (wouterjdb) [inline src/flownet/templates/webviz_ahm_config.yml.jinja2:7]:
```suggestion
  {%- for iteration in [iterations[0], iterations[-1]] %}
```

File: src/flownet/templates/webviz_ahm_config.yml.jinja2 [hunk 3]
```diff
@@ -14,22 +14,22 @@ pages:
     - The plot below show the results of the run for the different ensembles
     - HistoryMatch:
         ensembles:
-        {%- for iteration in iterations[-1:] %}
+        {%- for iteration in [iterations[0] , iterations[-1]] %}
```
Comment (wouterjdb) [inline src/flownet/templates/webviz_ahm_config.yml.jinja2:17]:
```suggestion
        {%- for iteration in [iterations[0], iterations[-1]] %}
```

File: src/flownet/templates/webviz_ahm_config.yml.jinja2 [hunk 4]
```diff
@@ -14,22 +14,22 @@ pages:
...
          - iter-{{ iteration }}
         {%- endfor %}
         observation_file: ./observations.yamlobs
  - title: Parameter distribution
    content: 
     - ParameterDistribution:
         ensembles:
-        {%- for iteration in iterations[-1:] %}
+        {%- for iteration in [iterations[0] , iterations[-1]] %}
```
Comment (wouterjdb) [inline src/flownet/templates/webviz_ahm_config.yml.jinja2:25]:
```suggestion
        {%- for iteration in [iterations[0], iterations[-1]] %}
```

File: src/flownet/templates/webviz_ahm_config.yml.jinja2 [hunk 5]
```diff
@@ -14,22 +14,22 @@ pages:
...
          - iter-{{ iteration }}
         {%- endfor %}
  - title: Reservoir simulation time series
    content:
     - ReservoirSimulationTimeSeries:
         ensembles:
-        {%- for iteration in iterations[-1:] %}
+        {%- for iteration in [iterations[0] , iterations[-1]] %}
```
Comment (wouterjdb) [inline src/flownet/templates/webviz_ahm_config.yml.jinja2:32]:
```suggestion
        {%- for iteration in [iterations[0], iterations[-1]] %}
```

**Comment by wouterjdb** [comment]
@tayloris As @anders-kiaer his status is still on "requested changes" he still needs to approve. But what now has been done is exactly what he wanted so it will just be a matter of him clicking the button. 👍

**Pull request merged**